### PR TITLE
Middle Mouse Control Point

### PR DESCRIPTION
**Pull request by Geevies**
* Control middle clicking now points at the object your mouse pointer is on.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Alberyk** [comment]
Fairly sure that the middle mouse button is already used by hardsuits.

**Comment by Geevies** [comment]
> 
> 
> Fairly sure that the middle mouse button is already used by hardsuits.

Since this is a && and not a ||, it *should* be fine, I think

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -89,19 +89,22 @@
 	var/list/modifiers = params2list(params)
 	if(modifiers["shift"] && modifiers["ctrl"])
 		CtrlShiftClickOn(A)
-		return 1
+		return TRUE
+	if(modifiers["shift"] && modifiers["middle"])
```
Comment (Arrow768) [inline code/_onclick/click.dm:93]: Sais shift here

File: html/changelogs/geeves-middle_control_mouse_point.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Control middle clicking now points at the object your mouse pointer is on."
```
Comment (Arrow768) [inline html/changelogs/geeves-middle_control_mouse_point.yml:6]: And control here.

**Pull request merged**